docs/user-guide.md

```diff
@@ -2702,13 +2709,268 @@ ok: [db-23ai-free]
 
 ## Post installation tasks
 
+### Validate the Oracle installation with ORAchk
+
+The orachk utility can be used to validate the Oracle installation. 
+
+ORAchk will check for known problems with the Oracle installation and configuration, and provide recommendations for resolving any issues that are found.
+
+To run ORAchk, download the AHF utility from the Oracle support site.
+
+The following Oracle Support Note will provide the download link:
+
+  `Autonomous Health Framework (AHF) - Including Trace File Analyzer and Orachk/Exachk (Doc ID 2550798.1)`
+
+ORAchk is a part of the AHF utility.  It is not necessary to install the entire AHF utility to run ORAchk.
+
+The script provided with the oracle-toolkit will allow you to install, run and uninstall ORAchk on the target server.
+
+#### The AHF Zip file.
+
+The zip file must be uploaded to GCS media bucket.
+
+The default 'directory' for this is AHF.
+
+For instance, the file `AHF-LINUX_v25.1.0.zip` would be uploaded to `gs://oracle-software/AHF/AHF-LINUX_v25.1.0.zip`.
+
+#### Install and Run ORAchk
+
+Use the the `check-oracle.sh` script to install and run ORAchk in one command.
+
+```bash
+./check-oracle.sh  --ahf-install \
+   --ora-swlib-bucket gs://pythian-gto-oracle-software \
+   --oracle-sid ORCL \
+   --oracle-server ora-db-server-orachk-19c-patch \
+   --ahf-dir AHF \
+   --ahf-file AHF-LINUX_v25.1.0.zip \
+   --run-orachk
+``` 
+
+example output:
+Note: skipped steps are omitted
+
+```text
+
+
+$ ./check-oracle-test.sh install-and-run
```

Comment:
This is coming as rather long:  300+ lines of sample output.  Can we skip some of the intermediate output just to show the command and some key results, so people don't have to scroll down too far through the user guide?

Reply:
examples shortened